filament/src/Renderer.cpp

```diff
@@ -252,17 +252,32 @@ void FRenderer::mirrorFrame(FSwapChain* dstSwapChain, Viewport const& dstViewpor
     // destination.
     driver.makeCurrent(dstSwapChain->getHwHandle(), mSwapChain->getHwHandle());
 
+    RenderPassParams params = {};
+    // Clear color to black if the CLEAR flag is set.
+    if ((flags & CLEAR) != 0) {
+        params.clear = TargetBufferFlags::COLOR;
+        params.clearColor = {0.f, 0.f, 0.f, 1.f};
+        params.left = 0;
+        params.bottom = 0;
+        driver.getMaxViewportDimensions(&params.width, &params.height);
```

Comment:
Why do we pass max viewport width & height here instead of the actual dimensions?

Reply:
@romainguy I think we don't have access to the SwapChain's dimensions here.

Reply:
yes that's the reason.  I'll switch it to max uint32_2 instead.